anitya/templates/project_new.html

```diff
@@ -119,10 +121,19 @@
     if (str == 'custom'){
       $('#regex_row').show();
       $('#version_url_row').show();
+      $('#regex').attr('placeholder', 'Leave empty for default regex');
     };
 
     $('#example_txt').html(examples[str]);
 
+    if (str == 'custom'){
+      $('#example_txt').append(
+        '<br /> More information at <a href="https://release-monitoring.org'
```

Comment:
Maybe use url_for, so that people get local links if they have their own setup of Anitya?